WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -82,6 +82,11 @@ - (instancetype)init
                                                  selector:@selector(helpshiftUnreadCountUpdated:)
                                                      name:HelpshiftUnreadCountUpdatedNotification
                                                    object:nil];
+
```

Comment:
+1 It'd be a good idea to add those removeObserver lines, just for the paranoia. 

Nevertheless, somewhere around iOS 4 or 5, Apple added a removeObserver call. Undocumented behavior, but if you try to crash an app that way, it won't break!.